.github/workflows/ci-actions-incremental.yml

```diff
@@ -304,7 +304,7 @@ jobs:
       - name: Build
         shell: bash
         # Despite the pre-calculated run_jvm flag, GIB has to be re-run here to figure out the exact submodules to build.
-        run: ./mvnw $COMMON_MAVEN_ARGS install -Dsurefire.timeout=1200 -pl !integration-tests/gradle -pl !integration-tests/maven -pl !integration-tests/devtools ${{ matrix.java.maven_args }} ${{ needs.build-jdk11.outputs.gib_args }}
+        run: ./mvnw $COMMON_MAVEN_ARGS install -Dsurefire.timeout=1200 -pl !integration-tests/gradle -pl !integration-tests/maven -pl !integration-tests/devtools -pl !docs ${{ matrix.java.maven_args }} ${{ needs.build-jdk11.outputs.gib_args }}
```

Comment:
Problem is we won't catch doc generation errors caused by a problem in the generated config doc references.

Or we need to trigger the separate `doc-build.yml` always and not only when adoc files are changed.

Reply:
> generated config doc references

Where/how those generated?

Reply:
They are generated when the extensions are built (by the quarkus-extension-processor IIRC).

Reply:
Well, the JVM Tests run with `skipDocs`, so isn't this all more or less deactivated?
See:
- https://github.com/quarkusio/quarkus/blob/main/.github/workflows/ci-actions-incremental.yml#L45 (used for all jobs, not only the JVM Tests)
- https://github.com/quarkusio/quarkus/blob/614b831eda3e284f8a49499bec35db5e3ed40550/core/processor/src/main/java/io/quarkus/annotation/processor/ExtensionAnnotationProcessor.java#L82
- https://github.com/quarkusio/quarkus/blob/614b831eda3e284f8a49499bec35db5e3ed40550/core/processor/src/main/java/io/quarkus/annotation/processor/ExtensionAnnotationProcessor.java#L245

Reply:
So... you're right... and that doesn't explain how this module could be a problem given it doesn't do anything noticeable (the asciidoc generation is skipped too) :).

Reply:
It's always in the jar-plugin, AFAICS. Is anyone consuming that jar? I don't think so?

Reply:
(Besides, given that no actual generation is performed in any of those JVM Test builds only underlines that it's useless to include `docs` in those jobs, in my opinion).

Reply:
I agree. I just don't understand WHY it would fail doing pretty much nothing :).

Reply:
Yeah, count me in on that!